pmd-java/src/test/java/net/sourceforge/pmd/lang/java/symboltable/AcceptanceTest.java

```diff
@@ -100,11 +100,10 @@ public void testDemo() {
         ASTMethodDeclaration node = acu.findDescendantsOfType(ASTMethodDeclaration.class).get(0);
         Scope s = node.getScope();
         Map<NameDeclaration, List<NameOccurrence>> m = s.getDeclarations();
-        for (Iterator<NameDeclaration> i = m.keySet().iterator(); i.hasNext();) {
-            NameDeclaration d = i.next();
-            assertEquals("buz", d.getImage());
-            assertEquals("ArrayList", ((TypedNameDeclaration) d).getTypeImage());
-            List<NameOccurrence> u = m.get(d);
+        for (Map.Entry<NameDeclaration, List<NameOccurrence>> entry : m.entrySet()) {
+            assertEquals("buz", entry.getKey().getImage());
+            assertEquals("ArrayList", ((TypedNameDeclaration) entry.getKey()).getTypeImage());
+            List<NameOccurrence> u = m.get(entry.getKey());
```

Comment:
This is wrong, should be `entry.getValue()`

Reply:
Fixed